main.go

```diff
@@ -64,6 +65,12 @@ func main() {
 		log.Fatal(err)
 	}
 
+	go func(w *watcher.FSWatcher) {
+		for range w.Event {
+			ex.Send(rebuild, nil)
```

Comment:
yeah, I guess you're right. On that same note it'd make it so the UI would work the same way as this... in any case we should sit down and think about how to proceed.